docs/modules/ROOT/pages/documentation/ecosystem/maven-plugin.adoc

```diff
@@ -317,6 +320,43 @@ the process id of the running Payara Micro instance.
 
 |===
 
+=== `reload`
+*Aim*: This goal reloads the Payara Micro application with the specified configuration.
+By default, this goal performs the vanilla deployment of the application without restarting the 
+Payara Micro instance. If `hotDeploy` is set to true then the Hot Reload functionality
+ will be enabled. 
+
+NOTE:
+This goal is specifically designed for integration with the IDEs and Editors.
+Payara Micro instance allows IDEs to deploy the application in Hot Deploy mode by reusing
+the existing application instance and updating its classloader & internal components
+relative to the modified source. This can lead to significant savings of time when
+an application is redeployed and boost developer productivity even more.
+
+*Usage*: `mvn payara-micro:reload`
+
+.Configuration tags
+[cols="2,1,7",options="header"]
+|===
+|Element
+|Default
+|Notes
+
+|`hotDeploy`
+|false
+|Enables the hot reload mode.
+
+|`sourcesChanged`
+|_none_
+|Defines a list(comma separated) of the absolute paths to the source file which need
+to be reloaded via the Hot Reload feature.
```

Comment:
```suggestion
to be reloaded via the Hot Deploy feature.
```
Although the name Hot Reload would make more sense here, we need to be consistent. Again, I'm not sure whether this is the same feature as the Hot Deploy feature available in Payara Micro and Payara Server and in IDEs, but if it is, then it should have the same name.